com.playeveryware.eos/Runtime/Core/Config/PlatformConfig.cs

```diff
@@ -266,6 +266,50 @@ protected PlatformConfig(PlatformManager.Platform platform) :
             Platform = platform;
         }
 
+        protected override void OnReadCompleted()
+        {
+            base.OnReadCompleted();
+
+            // If the deployment and client credentials are complete, there is
+            // nothing to do.
+            if (deployment.IsComplete && clientCredentials.IsComplete)
```

Comment:
This can get a nullref if `clientCredentials` is null, which is the default state of a new config. There needs to be a null check here.

Reply:
Yep - good catch. I've updated the implementation to first check for whether it is null.